src/caret_analyze/value_objects/message_context.py

```diff
@@ -70,6 +70,23 @@ def __init__(
         publisher: PublisherStructValue | None,
         child: tuple[CallbackStructValue, ...] | None,
     ) -> None:
+        """
+        Get node name.
+
+        Parameters
+        ----------
+        node_name : str
+            Node name.
+        message_context_dict : dict
+            Message context dict.
+        subscription : SubscriptionStructValue | None
+            Target subscription value.
+        publisher : PublisherStructValue | None
+            Target publisher.
+        child : tuple[CallbackStructValue, ...] | None
+            Child.
```

Comment:
`Childs.` or `Child elements.`

If there is a doc string you refer to, you do not have to change it.